libs/portals/admin/form-system/src/components/MainContent/components/Preview/components/MessageWithLink.tsx

```diff
@@ -0,0 +1,54 @@
+import { FormSystemInput } from '@island.is/api/schema'
+import { Box, Button, Text } from '@island.is/island-ui/core'
+
+interface Props {
+  data: FormSystemInput
+}
+
+export const MessageWithLink = ({ data }: Props) => {
+  const formatUrl = (url: string): string => {
+    if (url.startsWith('http://')) {
+      url = url.replace('http://', 'https://')
+    } else if (!url.startsWith('https://')) {
+      url = 'https://' + url
+    }
+    return url
+  }
+  return (
+    <Box
+      flexDirection={'row'}
+      display={'flex'}
```

Comment:
When providing string value to attributes please do not wrap in an object. Please review this in all your markup related files.
```suggestion
      flexDirection="row"
      display="flex"
```